backend/shork/src/main/kotlin/Settings.kt

```diff
@@ -12,10 +12,22 @@ import software.shonk.interpreter.internal.instruction.Dat
  *   interpreter stops. -1 for no limit
  */
 class Settings(
-    val coreSize: Int,
-    val instructionLimit: Int,
-    val initialInstruction: String,
-    val maximumTicks: Int,
+    /** The size of the core, in number of instructions it will fit. */
+    val coreSize: Int = 8092,
+    /**
+     * The maximum number of instructions will be executed before the interpreter stops and a draw
+     * is called.
+     */
+    val instructionLimit: Int = 100000,
+    /** The initial instruction the core will be filled with. */
+    val initialInstruction: String = "DAT #42, #1337",
```

Comment:
For compatibility, we should use the default "DAT $0, $0" instruction. Programs might use the a-field and b-field for operation, so it is a more suitable default instruction.